en/core-api/e-wallet.md

```diff
@@ -302,49 +313,69 @@ You will get the **API response** like the following.
+| transaction_time   | The date and time at which the transaction occurred.         | String | It is in the format, *YYYY-MM-DD* *HH:MM:SS.*<br>Time zone: Western Indonesian Time (GMT+7) |
+| transaction_status | The status of the transaction.                               | String | For more details, refer to [Transaction Status](/en/after-payment/get-status.md#transaction-status). |
+| fraud_status       | The fraud_status of the transaction is displayed.            | String | --                                                           |
+| actions            | The stage of the transaction                                 | Array  | There are four stages of the transaction that can be retrieved through this attribute: QR code, Deeplink, Status of transaction, Canceling the transaction |
+| name               | The name of the action.                                      | String | --                                                           |
+| method             | The type of the method.                                      | String | --                                                           |
+| url                | The redirect url                                             | String | --                                                           |
 
-The user flow varies when using a web browser (on a computer or a tablet) compared to a SmartPhone.
+## 2. Altering Payment Flow Depending on the Device Used
 
-### Show QR Code Image (on computer or tablet)
+ The user flow is different for web browser (on a computer or a tablet) and smartphone.
 
-To display transaction QR Code image, use the url from `generate-qr-code` actions retrieved from API response. Simplest way is to **"hotlink"** the image url, if the frontend is HTML, put the url in image tag `<img src="[QR CODE URL]">`, or display it on a similar component without downloading.
-If the frontend does not support such scenario, download the QR code image from that url, then display it on frontend.
+**Displaying *QR Code Image* (on computer or tablet)**
 
-Instruction Example for **QR Code** :
+The steps to be followed to display the QR code image:
 
-1. Tap **Pay using GoPay**
-2. QR code will appear on the next page
-3. Open any **QRIS compatible app** (e.g: Gojek) on your mobile phone
-4. Tap **Pay** then scan the QR Code
-5. Check and verify your payment details then tap **PAY**
-6. Complete your security PIN / verification
-7. Your transaction is finished
+1. URL`generate-qr-code`should be used. `generate-qr-code` can be derived from 'actions' (API Response).
+2. To hotlink the image URL is the simplest way to get the QR code image. In HTML frontend, you can put the URL in image tag (img src="[QR CODE URL]">) and directly get the image.
+3. If the frontend is other than HTML, the QR code image should be downloaded from the URL, and should be displayed on the frontend.
```

Comment:
Broken formattings, please fix

Reply:
> To hotlink the image URL is the simplest way to get the QR code image.
The wording that you modified from the original make it less understandable, can you improve?